.github/workflows/ci.yml

```diff
@@ -101,3 +161,59 @@ jobs:
         uses: actions-rs/cargo@v1.0.3
         with:
           command: test
+
+  tests_macos:
+    name: Run tests macos
+    runs-on: macos-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2.3.4
+
+      - name: Install Rust stable toolchain
+        uses: actions-rs/toolchain@v1.0.7
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+
+      - name:                      Rust Cache
+        uses:                      Swatinem/rust-cache@v1.3.0
```

Comment:
I'm curious; I've seen this spacing between the prop and value elsewhere for these cache lines and wasn't sure why; is there a reason for it?

Reply:
haha, we got a PR from the CI and he did this way but I can fix it ^

Reply:
Aah; I think I removed the spaces when I copied some CI bits from jsonrpsee but I've been curious ever since :D Makes sense!